website/src/data/companies.json

```diff
@@ -0,0 +1,64 @@
+{
+  "ABN AMRO": "https://www.abnamro.com",
```

Comment:
the structure of this file can be modified. for now i kept it simple with `{ \<name>: \<url> }, but we could also do something like:

```
{
  <name>: {
    url: <url>,
    src: <path/to/logo.svg>,
    // other fields?
  },
  ...
}
```

Reply:
I prefer the `name: {url, src}` style.